src/ingestion/calendar/census_api/client.py

```
@@ -0,0 +1,111 @@
+        if not isinstance(body, list) or len(body) < 2:
+            return []
+        headers = body[0]
+        if not isinstance(headers, list):
+            raise CensusEITSResponseError("Census EITS header row is not a list")
+        rows: list[dict[str, str]] = []
+        for raw_row in body[1:]:
+            if not isinstance(raw_row, list):
+                continue
+            rows.append({
+                str(key): str(value)
+                for key, value in zip(headers, raw_row)
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)